### PR TITLE
Output and feedback enhancements

### LINKED ISSUE #73: Add spinner for sites:list
Users with lots of sites can take a while to load.  Improve performance of this command and or add a spinner.

### PR DESCRIPTION
**Pull request by satyarohith**
**- Summary**
- add `-m` shorthand flag for `init` command
- add loading message to make `sites:list` feel faster

**- Test plan**

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**
🐎

closes #73

### CODE REVIEW

File: src/commands/sites/list.js [hunk 1]
```diff
@@ -1,15 +1,18 @@
 const { flags } = require('@oclif/command')
 const chalk = require('chalk')
+const ora = require('ora')
```
Comment (bcomnes) [inline src/commands/sites/list.js]:
```suggestion
const {cli} = require('cli-ux')
```

Comment (bcomnes) [reply src/commands/sites/list.js]: Lets use cli-ux here.

File: src/commands/sites/list.js [hunk 2]
```diff
@@ -1,15 +1,18 @@
...
 const Command = require('../../base')
 const renderShortDesc = require('../../utils/renderShortDescription')
 
 class SitesListCommand extends Command {
   async run() {
     const { flags } = this.parse(SitesListCommand)
     const { api } = this.netlify
+    const spinner = ora('Loading your sites').start()
```
Comment (bcomnes) [inline src/commands/sites/list.js]:
```suggestion
    cli.action.start('Loading your sites')
```

File: src/commands/sites/list.js [hunk 3]
```diff
@@ -1,15 +1,18 @@
...
     await this.authenticate()
 
     const sites = await api.listSites({ filter: 'all' })
+    spinner.stop()
```
Comment (bcomnes) [inline src/commands/sites/list.js]:
```suggestion
    cli.action.stop()
```

**Comment by satyarohith** [comment]
Updated! :tada:  

> I've had it on my wishlist to audit every async step to add spinners to indicate network activity.

I will update to this PR.

**Comment by satyarohith** [comment]
@bcomnes `cli-ux` persists the loading message which is not so appealing in few cases,
I would like to know your opinion on using `ora`  for spinners and also on using `opn`(great cross-platform support) to open links in the browser because `cli-ux` fails to all the time in Linux.

**Comment by bcomnes** [comment]
@satyarohith Mainly consistency with oclif recommendations, but I wasn't aware of those issues.  Let me look into it some more.

**Comment by bcomnes** [comment]
Triaging this onto my todo again

**Comment by bcomnes** [comment]
I can review this again tomorrow.

**Comment by bcomnes** [comment]
Thank you @satyarohith I'll it.

**Pull request merged**